### ISSUE TITLE
Automatically maximise / activate VS Code when breakpoint hit

### ISSUE THREAD

**Issue by garygreen** [Microsoft/vscode#46198]
The js debugger does this automatically however the php debugger by felix does not. So it seems like it's not a built in option in VS Code, would it make sense to have it as a config option?

```json
"debug.autoOpenWhenBreakpointHit": true
```

**Comment by isidorn** [comment]
The vscode window flashes when a breakpoint is hit. So closing this as desgined behavior.
If vscode is not active when a breakopint is hit an dthe window does not flash for you then please file a bug

**Comment by garygreen** [comment]
@ishanarora I believe you misunderstood. I realise the flashing window is designed and that WORKS. What I'm suggesting in this issue is to MAXIMISE / OPEN / SHOW vscode automatically on the debugger screen when a breakpoint is hit. The node debugger does this automatically, so I believe it is plugin specific - would it make sense to have this feature built into the CORE as an option?

**tldr;** During debugging, when a breakpoint is hit I'm sure developers would appreciate the ability for VSCode to automatically show, without having to manually open VSCode to see the hit breakpoint.

**Comment by isidorn** [comment]
I doubt we will add this as a CORE option so I am reluctant to reopen this as a feature request.
But thank you for your suggestion

**Comment by garygreen** [comment]
@isidorn ok no worries. Is it possible you could point me in the right direction of how I can achieve doing this in a plugin? I'd imagine it'd be pretty simple, listening for a breakpoint event and calling some api to maximise the window / show the debugger tab?

**Comment by isidorn** [comment]
@garygreen currently we do not have extension api to maximize the window so that is missing.
Also stop events are not expossed in the API. Bottom line this is not possible atm
